src/components/data/data-utils.js

```diff
@@ -152,8 +152,10 @@ export const EntityDeleter = ({ onDismiss, onSuccess, namespace, name, selectedE
   const [additionalDeletions, setAdditionalDeletions] = useState([])
   const [deleting, setDeleting] = useState(false)
 
+  const selectedKeys = _.keys(selectedEntities)
+
   const doDelete = async () => {
-    const entitiesToDelete = _.concat(_.map(entityName => ({ entityName, entityType: selectedDataType }), selectedEntities), additionalDeletions)
+    const entitiesToDelete = _.concat(_.map(entity => ({ entityName: entity.name, entityType: entity.entityType }), selectedEntities), additionalDeletions)
```

Comment:
This might be a clearer way to do this, leading people through the operations in order and making the map on the entity objects a bit clearer: 

```suggestion
    const entitiesToDelete = _.flow(
      _.map(({ name: entityName, entityType }) => ({ entityName, entityType })),
      entities => _.concat(entities, additionalDeletions)
    )(selectedEntities)
```

That third line with the arrow function can just be `_.concat(additionalDeletions)` if the order of deletions doesn't matter.